functionnal_tests/testing_file

```diff
@@ -0,0 +1,85 @@
+package controllers
```

Comment:
rename it with a `.go` 😉

Reply:
I can not rename it with .go extension because go will compile it with test and fail test